specs/protocol/deputy-pause-module.md

```diff
@@ -0,0 +1,439 @@
+- [aDPM-001](#adpm-001-contract-is-configured-correctly)
+
+### iSCP-002: Superchain-wide pause is not activated outside of the standard process
+
+We must maintain that the Superchain-wide pause is not activated outside of the standard process
+that approves the usage of the pause.
+
+#### Impact
+
+**Severity: High**
+
+If this invariant is broken, all components that rely on the Superchain-wide pause would be placed
+into a paused state unexpectedly. This would cause a temporary liveness failure for withdrawals
+through the Standard Bridge system and would negatively impact users until liveness could be
+restored by either the Guardian or the Deputy Guardian.
+
+#### Dependencies
+
+- [aSCP-001](#ascp-001-superchain-wide-pause-authorization-process-is-well-defined)
+- [iDPM-001](#idpm-001-only-the-deputy-may-act-through-the-module)
+- [iDPM-004](#idpm-004-deputy-authorizations-must-not-be-replayable)
+- [aDPM-002](#adpm-002-deputy-key-is-not-compromised)
+
+## Component Invariants
+
+### iDPM-001: Only the Deputy may act through the module
+
+The Deputy account must be the only address that can act through the module. Other accounts can
+execute an action on behalf of the Deputy if the account has a valid authorization signature from
+the Deputy for that action. No account may act through the module other than with the explicit
+authorization of the Deputy.
+
+#### Impact
+
+**Severity: High**
+
+If this invariant is broken, accounts other than the Deputy would be able to carry out the actions
+allowed by this module. Assuming that
+[iDPM-002](#idpm-002-deputy-must-only-be-able-to-trigger-the-superchain-wide-pause) holds, this
+means that an account other than the Deputy would be able to trigger the Superchain-wide pause,
+presumably without the authorization of the social consensus process that typically triggers this
+pause.
+
+#### Mitigations
+
+- Signature verification on the pause action.
+- Monitoring for pause triggering actions.
+
+#### Dependencies
+
+- [aDPM-006](#adpm-006-openzeppelin-ecdsa-and-eip712-contracts-are-free-of-critical-bugs)
+
+### iDPM-002: Deputy must only be able to trigger the Superchain-wide pause
+
+The Deputy must only be able to trigger the Superchain-wide pause action by causing the module to
+call the `pause` function on the `DeputyGuardianModule`. The Deputy must not be able to trigger any
+other action.
+
+#### Impact
+
+**Severity: Critical**
+
+If this invariant is broken, the Deputy would be able to cause the Optimism Foundation Operations
+Safe to execute some unknown set of possible actions. We would treat this as a critical risk.
+
+#### Mitigations
+
+- Strict auditing and verification of the `DeputyPauseModule`.
+
+#### Dependencies
+
+- [aDPM-005](#adpm-005-deputyguardianmodule-pause-function-correctly-triggers-pause)
+- [aDPM-007](#adpm-007-safe-contracts-are-free-of-critical-bugs)
+
+### iDPM-003: Deputy must always be able to act through the module
```

Comment:
Covered in the [DeputyPauseModule_Pause_Test](https://github.com/ethereum-optimism/optimism/blob/sc/deputy-pause-module/packages/contracts-bedrock/test/safe/DeputyPauseModule.t.sol#L377) happy path fuzzing.